vmdb/app/views/report/_report_info.html.haml

```diff
@@ -116,7 +116,7 @@
               = s.v_zone_name
   %h3
     = _('Widgets')
-  - if @temp[:widget_nodes].nil? || (@temp[:widget_nodes] && @temp[:widget_nodes].empty?)
+  - if @widget_nodes.nil? || (@widget_nodes && @widget_nodes.empty?)
```

Comment:
if `@widget_nodes` is either nil, hash or array, can't this be simplified to `if @widget_nodes.blank?`